src/ext/java/org/opentripplanner/ext/fares/model/Distance.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.ext.fares.model;
+
+public class Distance {
```

Comment:
This class should be an immutable value type. Can you apply this commit which does exactly this? https://github.com/leonardehrenfried/OpenTripPlanner/commit/71025427efcdd5356d3678bb1f937115b2795d6f

Also please write a bit of Javadoc.

We need to speak to @t2gran where to put this class since we should start using it in other places.

Reply:
Is there already a decision on the placement or will it be moved possibly in later PRs?

Reply:
Thomas will need to review this but he is a busy man. I'm ok with merging this as is and figure out the correct place later.

Reply:
What about `org.opentripplanner.framework.scalars` ?

Reply:
If moved into core it need `hashCode()` and `equals()` (based on a normalized value). I suggest normalizing it in the constructor to the closest deci-meters (~1.2m). Optional you may use other units if larger than 1000m like `1.2 km` - but only SI units. All value objects end up in logs, tests and debugging and should not be translated into current locale. The `toString()` should include unit: Min. requirements for  `toString` would be -> "1,234.0m". Use `OtpNumberFormat#formatNumber(Number value, String unit)`  - this is a ValueObject so no class name or  '{}' should be printed.